README.md

```diff
@@ -17,7 +17,7 @@ $ npm install metasync
 
 ## Functional Asyncronous Composition
 
-Syntax: `metasync.composition(functions, [done, [data]]);`
+Syntax: `metasync.flow(functions)(data, callback);`
```

Comment:
Can you please add documentation in PRs that introduce the corresponding functionality for the ease of review in the future? Thanks!